lib/react_on_rails/helper.rb

```diff
@@ -94,8 +94,7 @@ def react_component(component_name, options = {})
     end
 
     def load_pack_for_component(component_name)
```

Comment:
why is this part of the public helper?

there's no doc for this method `load_pack_for_component` 
or `generated_components_pack_path`

Reply:
It should be private member as it is used internally by `react_component` and `react_component_hash`